### PR TITLE
[Notify] App removed from JWT's

### PR DESCRIPTION
**Pull request by flypaper0**

### CODE REVIEW

**Comment by flypaper0** [comment]
On hold until Monday

**Review by llbartekll — APPROVED** [review]

**Review by llbartekll — APPROVED** [review]
can you verify before merging that after pulling from notify-source-of-truth the tests are passing?

**Pull request closed without merging**
